src/main/java/sc/fiji/bdvpg/bdv/projector/AccumulateMixedProjectorARGB.java

```diff
@@ -80,16 +79,14 @@ protected void accumulate(
 				continue;
 			}
 
-			String projectionMode = getProjectionMode( sourceIndex, sourceAndConverterToMetadata );
-
-			if ( projectionMode.equals( "Sum" ) )
+			if ( projectionModes[sourceIndex].equals( Projection.PROJECTION_MODE_SUM ) )
```

Comment:
It's a function which is supposed to be fast, right ? I'd change the ".equals(string)" comparison with an int:
`projectionModes[sourceIndex] == PROJECTION_MODE_SUM` where `PROJECTION_MODE_SUM` is a `static final int`

Reply:
see below.